fake-koji/src/main/java/org/fakekoji/http/ProjectMapping.java

```diff
@@ -102,4 +104,51 @@ String getProductOfProject(String project, List<String> productList, List<String
         }
         throw new ProjectDoesNotMatchException();
     }
+
+    List<String> getBuildArchesOfProject(String project) throws ProjectMappingException {
```

Comment:
Next to this method we need also getBuildArchesOfNVRA

Reply:
Shouldn't it be getBuildArchesOfNVR? without arches?